src/Select.js

```diff
@@ -886,8 +900,13 @@ export default class Select extends Component<Props, State> {
                 'aria-label': label,
                 role: 'group',
               }}
-              Heading={GroupHeading}
```

Comment:
I thought this prop should be passed through the `components` props, rather than directly.. is that correct?

Reply:
nvm, I figured it out.. this was correct.